Makefile

```diff
@@ -35,7 +35,7 @@ docker-push:
 	docker push twilio/twilio-csharp:${CURRENT_TAG}
 
 cover:
-	dotnet sonarscanner begin /k:"$(PROJECT_NAME)" /o:"twilio" /d:sonar.host.url=https://sonarcloud.io /d:sonar.login="${SONAR_TOKEN}"  /d:sonar.language="cs" /d:sonar.exclusions="src/Twilio/Rest/**/*.*,test/Twilio.Test/**/*.*" /d:sonar.cs.opencover.reportsPaths="test/lcov.net451.opencover.xml"
+	dotnet sonarscanner begin /k:"$(PROJECT_NAME)" /o:"twilio" /d:sonar.host.url=https://sonarcloud.io /d:sonar.login="${SONAR_TOKEN}"  /d:sonar.language="cs" $(SONAR_SOURCES) /d:sonar.cs.opencover.reportsPaths="test/lcov.net451.opencover.xml"
```

Comment:
Prefer to have a default value defined in this file for `SONAR_SOURCES` which can be overridden. E.g., https://stackoverflow.com/a/24264930

Reply:
I think the same should go for the `PROJECT_NAME`.

Reply:
Added default values for both PROJECT_NAME and SONAR_SOURCES